lobid-vocab.ttl

```diff
@@ -256,9 +256,18 @@ lv:currentLocation
     rdfs:label "current location"@en ;
     rdfs:label "Standort"@de ;
     rdfs:isDefinedBy <http://purl.org/lobid/lv> ;
-    rdfs:range rdfs:Literal ;
-    rdfs:subPropertyOf dct:identifier .
+    rdfs:range rdfs:Literal .
 
+# <http://purl.org/lobid/lv#currentLibrary>
+
+lv:currentLibrary
+    a owl:DatatypeProperty ;
+    rdfs:comment "The specific library within an institution holding an item"@en ;
+    rdfs:domain bf:Item;
+    rdfs:label "current location"@en ;
+    rdfs:label "Standort"@de ;
```

Comment:
These labels haven't been adjusted after copy/paste.

Reply:
```suggestion
    rdfs:label "Current Location"@en ;
    rdfs:label "Standort"@de ;
```

Reply:
Like this?